geometry_msgs/Pose pose
---
string name
bool result